v3/integrations/nrgraphgophers/nrgraphgophers_test.go

```diff
@@ -244,7 +244,6 @@ func TestQueryRequestError(t *testing.T) {
 	app.ExpectErrors(t, []internal.WantError{{
 		TxnName: "WebTransaction/Go/POST /",
 		Msg:     "graphql: something went wrong",
-		Klass:   "*errors.QueryError",
```

Comment:
Why isn't this Klass returned in the error event anymore?

Reply:
that is a good question..

Reply:
Ah, I left myself a note in the commit message :smile:. The graphgophers library changed the error type that they return in a later version of this library. Because we test both a pinned version and the lates version, we need to be able to handle two different error types in this block of code, so I adjusted the logic to allow you to pass "" as the error klass which matches any error type. See the commit titled "Remove nrgraphgophers_test error type expectation" for this change.